project/plugins.sbt

```diff
@@ -7,9 +7,6 @@ addSbtPlugin("com.github.sbt"    %  "sbt-release"           % "1.0.15")
 addSbtPlugin("org.playframework" %  "sbt-plugin"            % "3.0.3")
 addSbtPlugin("uk.gov.hmrc"       %  "sbt-distributables"    % "2.5.0")
 addSbtPlugin("net.virtual-void"  %  "sbt-dependency-graph"  % "0.10.0-RC1")
-addSbtPlugin("org.scoverage"     %  "sbt-scoverage"         % "2.0.12")
-addSbtPlugin("org.scalastyle"    %% "scalastyle-sbt-plugin" % "1.0.0" exclude("org.scala-lang.modules", "scala-xml_2.12"))
-addSbtPlugin("com.timushev.sbt"  %  "sbt-updates"           % "0.6.3")
```

Comment:
Isnt this for dependency updates

Reply:
Yep removed wrong line. Added sbt-updates back in. and removed scala-xml.